rust_src/src/lread.rs

```diff
@@ -182,4 +184,56 @@ pub fn read(stream: LispObject) -> LispObject {
     }
 }
 
+/// Execute the region as Lisp code.
+/// When called from programs, expects two arguments,
+/// giving starting and ending indices in the current buffer
+/// of the text to be executed.
+/// Programs can pass third argument PRINTFLAG which controls output:
+///  a value of nil means discard it; anything else is stream for printing it.
+///  See Info node `(elisp)Output Streams' for details on streams.
+/// Also the fourth argument READ-FUNCTION, if non-nil, is used
+/// instead of `read' to read each expression.  It gets one argument
+/// which is the input stream for reading characters.
+///
+/// This function does not move point.
+#[lisp_fn(min = "2")]
+pub fn eval_region(
+    start: LispObject,
+    end: LispObject,
+    printflag: LispObject,
+    read_function: LispObject,
+) -> LispObject {
```

Comment:
Do we want to update this comment to reflect we have inherited it? Can we resolve it?

Reply:
I don't understand this comment :)

Reply:
The FIXME that you carried over. What should we do with it?

Reply:
I meant the FIXME.

Reply:
I still don't get it. `eval-sexp-add-defvars` is a lisp function ?

Reply:
Yes, it's in `elisp-mode.el`. `cconv.el` has some TODOs at the top:

    ;; TODO: (not just for cconv but also for the lexbind changes in general)
    ;; - let (e)debug find the value of lexical variables from the stack.
    ;; - make eval-region do the eval-sexp-add-defvars dance.

and so on. Since it's 8 years old, I guess it's not terribly important. I guess you can just leave the FIXME in place :)